src/ng-select/ng-select.component.ts

```diff
@@ -59,7 +59,8 @@ export type DropdownPosition = 'bottom' | 'top' | 'auto';
         '[class.top]': 'currentDropdownPosition === "top"',
         '[class.bottom]': 'currentDropdownPosition === "bottom"',
         '[class.ng-single]': '!multiple',
-        '[class.ng-selected]': 'hasValue'
```

Comment:
have you tried with `@HostBinding` decorator?

Reply:
yes, same "shit"

Reply:
Move this to ng-control instead of host class